SPECS/qemu/qemu.spec

```diff
@@ -1,64 +1,257 @@
-# Disable stripping - strip is trying to strip binaries (.img, .elf, etc.) built for different architectures.
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+# Provide a way to skip tests via rpmbuild `--without`
+# This makes it easier to skip tests in copr repos, where
+# the qemu test suite is historically flakey
+%bcond_without check
+
+# This spec is for AzLinux
```

Comment:
i agree with what you're doing here with the macros (and preserving the other distro macros), but i dislike having our specs follow a different set of rules than all of the other specs.  i've been following the mariner recommendations:
* removing all distro macros (fedora, rhel, suse, azl)
* alphabetizing and grouping the buildrequires, requires, etc

Reply:
I agree that the cleanup is needed since that is the current recommended strategy. I will do it towards the end of the review. Ideally, I would like to do it as a separate commit so the git history has the unmodified (fedora-like) version as well.

Reply:
Removed all references to fedora and rhel. For the BR the main set has been put in alphabetic order. There are a few misc conditional BRs that are separate. It is similar to how 2.0 is layout.